### ISSUE TITLE
THREE.WebGLProgram: shader error.when set renderer.shadowMap.enabled to true

### ISSUE THREAD

**Issue by KinoriN** [pixiv/three-vrm#693]
* error log 
``` 
three.module.js:17241 THREE.WebGLProgram: shader error:  0 35715 false gl.getProgramInfoLog Fragment shader is not compiled.  
```   
```  
renderer.shadowMap.enabled = true;
const light = new THREE.DirectionalLight(0xffffff);
      light.position.set(1.0, 1.0, 1.0).normalize();
      light.castShadow = true;  
```  

* code  
```  
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <title>three-vrm example</title>
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
    />
    <style>
      body {
        margin: 0;
      }
      canvas {
        display: block;
      }
    </style>
  </head>

  <body>
    <script src="https://unpkg.com/three@0.125.2/build/three.js"></script>
    <script src="https://unpkg.com/three@0.125.2/examples/js/loaders/GLTFLoader.js"></script>
    <script src="https://unpkg.com/three@0.125.2/examples/js/controls/OrbitControls.js"></script>
    <script src="../lib/three-vrm.js"></script>
    <script>
      // renderer
      const renderer = new THREE.WebGLRenderer();
      renderer.setSize(window.innerWidth, window.innerHeight);
      renderer.setPixelRatio(window.devicePixelRatio);
      renderer.shadowMap.enabled = true;
      document.body.appendChild(renderer.domElement);

      // camera
      const camera = new THREE.PerspectiveCamera(
        30.0,
        window.innerWidth / window.innerHeight,
        0.1,
        20.0
      );
      camera.position.set(0.0, 1.0, 5.0);

      // camera controls
      const controls = new THREE.OrbitControls(camera, renderer.domElement);
      controls.screenSpacePanning = true;
      controls.target.set(0.0, 1.0, 0.0);
      controls.update();

      // scene
      const scene = new THREE.Scene();

      // light
      const light = new THREE.DirectionalLight(0xffffff);
      light.position.set(1.0, 1.0, 1.0).normalize();
      light.castShadow = true;

      scene.add(light);

      // gltf and vrm
      const loader = new THREE.GLTFLoader();
      loader.crossOrigin = "anonymous";
      loader.load(
        // URL of the VRM you want to load
        "./models/three-vrm-girl.vrm",

        // called when the resource is loaded
        (gltf) => {
          // calling this function greatly improves the performance
          THREE.VRMUtils.removeUnnecessaryJoints(gltf.scene);

          // generate VRM instance from gltf
          THREE.VRM.from(gltf).then((vrm) => {
            console.log(vrm);
            scene.add(vrm.scene);

            vrm.humanoid.getBoneNode(
              THREE.VRMSchema.HumanoidBoneName.Hips
            ).rotation.y = Math.PI;
          });
        },

        // called while loading is progressing
        (progress) =>
          console.log(
            "Loading model...",
            100.0 * (progress.loaded / progress.total),
            "%"
          ),

        // called when loading has errors
        (error) => console.error(error)
      );

      // helpers
      const gridHelper = new THREE.GridHelper(10, 10);
      scene.add(gridHelper);

      const axesHelper = new THREE.AxesHelper(5);
      scene.add(axesHelper);

      function animate() {
        requestAnimationFrame(animate);

        renderer.render(scene, camera);
      }

      animate();
    </script>
  </body>
</html>

```

**Comment by 0b5vr** [comment]
Reproduced. It seems the shader (`mtoon.frag`) uses a code snippet from a very previous version of Three.js.

![image](https://user-images.githubusercontent.com/7824814/116350237-919dd700-a82c-11eb-99a7-565493ad0954.png)

Thanks for the report so far!

**Comment by 0b5vr** [comment]
will be shipped in the next update :bow:

**Comment by 0b5vr** [comment]
It's now shipped @ v0.6.4
